include/wx/filehistory.h

```diff
@@ -24,6 +24,13 @@ class WXDLLIMPEXP_FWD_CORE wxMenu;
 class WXDLLIMPEXP_FWD_BASE wxConfigBase;
 class WXDLLIMPEXP_FWD_BASE wxFileName;
 
+enum wxFileHistoryMenuLabelStyle
```

Comment:
Sorry for the first (but not last) of naming nitpicks, but I think it would be useful to use the word "path" here, and in the names of the functions dealing with this enum, because otherwise it's not obvious at all that it has something to do with them. Maybe just call it `wxFileHistoryMenuPathStyle`?

Reply:
Just FYI, I've renamed it now.